### DISCUSSION TITLE
“Same folder” plugin

### DISCUSSION THREAD

**Discussion by martid1703** [elfmz/far2l#1523] [Q&A] [answered]
Привет! Подскажите, есть ли в far2l стандартный плагин “Same folder”? Привык им пользоваться в Win версии.
Сделал временно макрос, и вроде он работает, но кажется он не оптимальный и хотелось бы в целом работать через общую точку вызова плагинов.
Макрос:
[KeyMacros/Shell/AltShiftS]
DisableOutput=0x1
Sequence=Tab Num7 AltShiftNum0 Tab c d Space ShiftNum0 Enter

**Comment by elfmz** [comment, marked as answer]
разве самый первый пункт (имеющий хоткей '-') в меню Alt-F1/F2 - не оно?

**Reply by martid1703** [reply]
точно, я так привык к F11+S(same folder) плагину, что даже не обратил внимания. Спасибо!
